README.md

```diff
@@ -0,0 +1,46 @@
+# Pawntastic
+An object-oriented game written in Java using libGDX.
+
+<!-- Badge showing test suite status -->
+[![Test Suite](https://github.com/martinjonsson01/OOPP-WITH-THE-BOIS/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/martinjonsson01/OOPP-WITH-THE-BOIS/actions/workflows/test.yml)
+
+## Understanding the Architecture
+There is a UML class diagram of each package located in the README file of each package directory. It is visible directly when navigating to the package directory on GitHub, but if viewing the files locally the raw images can be found in `documents/diagrams/`.
+
+![com.thebois package class diagram](documents/diagrams/com.thebois.jpg "com.thebois package class diagram")
+
+## Running the Game
+
+### Prerequisites
+
+* Java 15 or later
+
+To run the latest version of the game, download the jar file from [releases](https://github.com/martinjonsson01/OOPP-WITH-THE-BOIS/releases/) and run it using the command:
+```terminal
+java -jar ColonyManagement-vX.X.X.jar
+```
+
+## Running From Source
+
+To build the game directly from the source code, you need to have some common development tools installed.
+
+### Prerequisites
+
+* [JDK 15](https://jdk.java.net/archive/)
+* [Maven](http://maven.apache.org/install.html) installed according to instructions
```

Comment:
Nice to have the links here. But might want them for the prerequisites for running it from the release as well!

Reply:
Neither maven nor the JDK should be necessary for running the jar, the JRE should be enough. I'll add a link to the JRE.